corehq/apps/hqmedia/templates/hqmedia/partials/multimedia_uploader.html

```diff
@@ -27,9 +27,6 @@ <h3 class="modal-title">{% trans "Upload" %} {{ uploader.media_type|title }}</h3
           </label>
           <div class="hqm-existing-controls col-sm-9"></div>
         </div>
-        <div class="hqm-upload-form">
-          {% include 'hqmedia/partials/hqm_upload_form.html' %}
-        </div>
```

Comment:
Is multimedia_uploader entirely not in use anymore? but we are removing just a part of it?

Reply:
This just removes a part of it, although I'm working on removing it altogether.

Reply:
okay thank you. It was not clear to me how we determined this part was safe to remove. I thought we don't have uploaders anymore, but I could not determine that from the views.

Reply:
The uploader code is rather...layered. I'm still getting my head around it. The functionality removed in this particular PR is something I also removed in Vellum, [here](https://github.com/dimagi/Vellum/pull/1122/commits/48eb028dfdd363b8475bc130551e770f68a027db), so I just verified that it was safe to make the same change in HQ (and I think it's safe, just based on [this](https://github.com/dimagi/commcare-hq/pull/35181#discussion_r1790557417)).

Reply:
Cool, that is helpful :+1: